### PR TITLE
Adding merging type (DAQ) and changing defaults (81X)

### PR DESCRIPTION
**Pull request by smorovic**
Port of #14528 to 81X

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by mommsen** [comment]
+1

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/14543	5a716e5
- same code changes as in 81X
- offline reco code is not affected

**Comment by Martin-Grunewald** [comment]
+1

**Comment by dmitrijus** [comment]
+1

File: DQMServices/Components/src/DQMFileSaver.cc [hunk 1]
```diff
@@ -261,7 +261,7 @@ DQMFileSaver::saveForOnline(int run, const std::string &suffix, const std::strin
 
 
 boost::property_tree::ptree
-DQMFileSaver::fillJson(int run, int lumi, const std::string& dataFilePathName, const std::string transferDestinationStr, evf::FastMonitoringService *fms)
+DQMFileSaver::fillJson(int run, int lumi, const std::string& dataFilePathName, const std::string transferDestinationStr, const std::string mergeTypeStr, evf::FastMonitoringService *fms)
```
Comment (davidlange6) [inline DQMServices/Components/src/DQMFileSaver.cc]: perhaps worth making these const std::strings into const std::string&s in the future

**Comment by davidlange6** [comment]
+1

**Pull request merged**